korman/ui/ui_texture.py

```diff
@@ -38,6 +38,9 @@ def draw(self, context):
         layer_props = context.texture.plasma_layer
         layout = self.layout
 
+        layout.prop(layer_props, "envmap_color")
+        layout.separator()
```

Comment:
pointless UI ordering change?

Reply:
See the second point, and the full commit message:

> Reorder DEM properties to fix vanishing color prop
>
> Not sure why this was happening, but if you got into a state where no
VisRegion was selected, the Environment Clear Color property would also
disappear.
>
> Reordered to put the Clear Color at the top, so that it works
consistently and also makes it more obvious that it's unrelated to the
visibility regions.